task_sdk/tests/api/test_client.py

```diff
@@ -133,3 +133,33 @@ def handle_request(request: httpx.Request) -> httpx.Response:
                 "reason": "not_found",
             }
         }
+
+
+class TestXCOMOperations:
+    """
+    Test that the XComOperations class works as expected. While the operations are simple, it
+    still catches the basic functionality of the client for xcoms including endpoint and
+    response parsing.
+    """
+
+    # TODO: Add tests for get xcom in a follow up
```

Comment:
You can add it in this one too if you want.. should be a simple test

Reply:
Sure, adding in this PR itself

Reply:
Handled in 12fd0c8cdc1fb7675fa2d925fc2a008026dc7e45